tests/test_client/urls.py

```diff
@@ -5,6 +5,7 @@
 from . import views
 
 urlpatterns = [
+    url(r'^minimal_view/$', views.minimal_view, name='minimal_view'),
```

Comment:
I'm a bit surprised if there aren't any existing file upload tests we could reuse. Maybe in tests/test_client_regress if not here? It would be nice to test that the upload works, not just that no exceptions are raised.